http-transport-binding.md

```diff
@@ -330,6 +340,105 @@ Content-Length: nnnn
 
 ```
 
+### 3.3. Batched Content Mode
+
+In the *batched* content mode several events are batched into a single HTTP
+request or response body. The chosen [event format](#14-event-formats) MUST
+define how a batch is represented. Currently, the only format supporting
+batching is the [JSON Batch Format][JSON-batch-format].
+
+#### 3.3.1. HTTP Content-Type
+
+The [HTTP `Content-Type`][Content-Type] header MUST be set to the media type of
+an [event format](#14-event-formats).
+
+Example for the [JSON Batch format][JSON-batch-format]:
+
+``` text
+Content-Type: application/cloudevents-batch+json; charset=UTF-8
+```
+
+#### 3.3.2. Event Data Encoding
+
+The chosen [event format](#14-event-formats) defines how a batch of events and
+all event attributes, including the `data` attribute, are represented.
+
+The batch of events is then rendered in accordance with the event format
+specification and the resulting data becomes the HTTP message body.
+
+The batch MAY be empty (typically used in a HTTP response).
```

Comment:
Strike "(typically used in a HTTP response)". There might be reasons to push an empty (zero elements) batch.